### PR TITLE
Export calendar

### PR DESCRIPTION
**Pull request by datanel**
# Description

needs https://github.com/CanalTP/NmmPortalBundle/pull/24

This PR add the export of calendars with a zip containing 2 csv (grid_calendars.csv and grid_periods.csv)

# Pull Request type

new feature

# How to test

In the bundle
```
composer update
./vendor/bin/phpunit
```

In Timetable go to /mtt/calendars/list, create calendars and click to export button

# Team reviewers

@alcalyn @asfram

### CODE REVIEW

File: Resources/views/Calendar/list.html.twig [hunk 1]
```diff
@@ -5,6 +5,9 @@
 {% block list_title %}{{ 'calendar.list.title'|trans }}{% endblock %}
 {% block list_menu %}
     <div class="text-right">
+        <a {% if not calendars %}disabled"{% endif %} id="calendar_export_btn" class="btn btn-primary" href="{{ path('canal_tp_mtt_calendars_export') }}">
```
Comment (stanosz) [inline Resources/views/Calendar/list.html.twig]: the disabled function not working in link balise

**Comment by datanel** [comment]
Jenkins can you test this please?

**Comment by datanel** [comment]
Jenkins can you test this please?

**Comment by datanel** [comment]
Jenkins can you test this please?

**Comment by datanel** [comment]
Jenkins can you test this please?

**Comment by datanel** [comment]
Jenkins can you test this please?

**Comment by datanel** [comment]
Jenkins can you test this please?

**Comment by datanel** [comment]
Jenkins can you test this please?

**Pull request merged**